feature/onboarding/welcome/src/main/kotlin/app/k9mail/feature/onboarding/welcome/ui/WelcomeContent.kt

```diff
@@ -75,13 +84,22 @@ private fun WelcomeLogo(
     modifier: Modifier = Modifier,
 ) {
     Column(
-        modifier = Modifier.then(modifier),
+        modifier = modifier,
         horizontalAlignment = Alignment.CenterHorizontally,
     ) {
-        Image(
-            painter = painterResource(id = R.drawable.onboarding_welcome_logo),
-            contentDescription = null,
-        )
+        Box {
+            Canvas(modifier = Modifier.size(CIRCLE_SIZE_DP.dp)) {
+                drawCircle(color = Color(CIRCLE_COLOR))
+            }
```

Comment:
The same could be achieved by using `clip` modifier and `CircleShape`

```suggestion
        Box(
            modifier = Modifier
                .clip(CircleShape)
                .background(Color(CIRCLE_COLOR))
                .size(CIRCLE_SIZE_DP.dp),
        ) {
```